src/main/resources/reference.conf

```diff
@@ -20,20 +20,16 @@ resourceTypes =
     },
     {
       name = "billing-project"
-      actions = ["create_workspace", "alter_policies", "read_policies", "launch_batch_compute", "launch_notebook"]
+      actions = ["create_workspace", "alter_policies", "read_policies", "launch_batch_compute", "launch_notebook", "can_share_.+"]
       ownerRoleName = "owner"
       roles = [
         {
           roleName = "owner"
           roleActions = ["create_workspace", "alter_policies", "read_policies", "launch_batch_compute", "launch_notebook"]
         },
-        {
-          roleName = "user"
-          roleActions = ["create_workspace"]
-        },
         {
           roleName = "workspace-creator"
-          roleActions = ["create_workspace"]
+          roleActions = ["create_workspace", "can_share_can-compute-user"]
```

Comment:
I like this because it is an easy way to give all users with workspace-creator role the can share action, no migration required. I don't like it because can-compute-user is a policy name that is defined in rawls.

Reply:
Can `can-compute-user` be thought of as being defined here and referenced in rawls?